datasets/modeled/industry/dataset.json5

```
@@ -53,14 +52,14 @@
       class: 'ModelYear',
       name: 'model_year',
       description: 'IEF 2023 Model Years; Biennial 2024-2030, then every 5 years to 2050',
-      file: "../../../project/dimensions/model_years.csv",
+      file: "dimensions/model_years.csv",
       module: "dsgrid.dimension.standard",
     },
     {
       type: "scenario",
       class: "Scenario",
       name: "gcam_scenario",
-      description: "IEF Year 1 Task 4 GCAM Scenarios",
+      description: "IEF Year 2 Task 2 GCAM Scenarios",
```

```suggestion
      description: "IEF Year 2, Task 2.2 GCAM Scenarios",
```
@sjsmith11, I know we've been calling our task "IEF Task 2.1," so I'm guessing that the GCAM task should be "Task 2.2," but is that your understanding as well?